### ISSUE TITLE
Extension does not display existing tags on web app

### ISSUE THREAD

**Issue by jbeldia** [sn-extensions/autocomplete-tags#11]
On the web app (Chrome 83.0.4103.97), a user reported that the extension doesn't suggest existing tags anymore. With the extension deactivated, they still see the current tags on their note:

![quick tags error 1](https://user-images.githubusercontent.com/54989748/84776861-71e63180-b013-11ea-811c-990322d1522c.png)

Once they activate the extension, the current tags are no longer there, and when they attempt to type in an existing tag, it suggests to create a new tag instead, and the dropdown gets cutoff awkwardly at the bottom:

![quick tags error 2](https://user-images.githubusercontent.com/54989748/84777146-cbe6f700-b013-11ea-950e-a46e1fdf8e2c.png)
![quick tags error 3](https://user-images.githubusercontent.com/54989748/84777155-ce495100-b013-11ea-87b8-a9cff1a9d7c6.png)

Reinstalling Quick Tags didn't fix the issue, and neither did signing out and back into their account.

I was able to reproduce this behavior on Chrome, but only on a test account that had a fresh install of the Quick Tags extension.

**Comment by mobitar** [comment]
I can't replicate this. You get similar behavior if after activating Quick Tags you ignore the permissions window. Do you see this window and press Continue on it?

<img width="350" alt="Screen Shot 2020-06-16 at 9 37 21 AM" src="https://user-images.githubusercontent.com/3277844/84788726-0f5e4880-afb5-11ea-95da-1c65e2855b76.png">

**Comment by jbeldia** [comment]
Yup, the activation window did appear, and I was sure I clicked Confirm!

I tried it out again a while ago by reinstalling Quick Tags on my test account, sadly I still get the same behavior :(

**Comment by JaspalSuri** [comment]
Does the console provide any helpful information?

**Comment by jbeldia** [comment]
@JaspalSuri Thanks for the tip, I checked the console and I found some error messages mentioning that there was some difficulty loading the other extensions, so I tried refreshing the repository in my test account. 

It didn't fix the issue right away, but after I signed out and back into my account one last time, I noticed that a conflicting copy of a tag appeared on my sidebar. After I deleted the conflicting copy of the tag, the extension started working again! I'll try to see if it's the same case for the user that originally reported this!
